spec/system/onboardings/user_completes_onboarding_spec.rb

```diff
@@ -2,19 +2,21 @@
 
 RSpec.describe "Completing Onboarding", type: :system, js: true do
   let(:password) { Faker::Internet.password(min_length: 8) }
-  let(:user) { create(:user, password: password, password_confirmation: password, saw_onboarding: false) }
 
-  after do
-    sign_out user
+  before do
+    allow(SiteConfig).to receive(:allow_email_password_registration).and_return(true)
+    allow(SiteConfig).to receive(:allow_email_password_login).and_return(true)
```

Comment:
Since this spec only uses email/password for logging in, we need to mock these. I suspect this was causing the flakiness but I really don't know for sure.

Reply:
Hey @vaidehi, I actually support your approach to fixing the flakiness; in my previous work on `/admin/config` my specs would keep failing intermittently  if I hadn't defined some Config settings like `mascot image` in my spec file